mappings/338c17dffaaefdb97ace91100724836178c3f9dd994a4798a66f546d4d414e4e59.json

```diff
@@ -22,7 +22,7 @@
     },
     "decimals": {
         "sequenceNumber": 0,
```

Comment:
Hi @TruthSeeker1987,

you should increment sequenceNumber for the updated values. 

[How do I update my entry in the registry?](https://github.com/cardano-foundation/cardano-token-registry/wiki/How-do-I-update-my-entry-in-the-registry%3F)

Thanks!

Reply:
Thank you so much for your input.  I REALLY appreciate it.  Just to clarify the situation.  On min swap and wingriders it shows that I only have 99,000 tokens instead of 9,990,000 in my wallet (not sure why but Sundaeswap displays correctly).  I am trying to correct the situation.  So you are saying that for "sequenceNumber" I should put 4 or 0 to correct the problem.  Again, I genuinely appreciate the help.

Reply:
Let me clarify, if I make or alter the token in a token creator so that it is correct.  In the blank that it asks for "decimal" should I put 0 or 4?  Thx

Reply:
Okay.  I have made changes that I believe should fix the problems.  Thank you again for all your work and I hope this works flawlessly.